src/node_crypto.cc

```diff
@@ -391,11 +394,15 @@ void ThrowCryptoError(Environment* env,
       .ToLocalChecked();
   CryptoErrorVector errors;
   errors.Capture();
-  Local<Value> exception = errors.ToException(env, exception_string);
+  Local<Value> exception;
+  MaybeLocal<Value> exception_maybe = errors.ToException(env, exception_string);
+  if (!exception_maybe.ToLocal(&exception))
```

Comment:
Fwiw, the most prevalent style to inline the checked call into the `if()`, i.e. `if (!errors.ToException(...).ToLocal(&exception)) ...` and skipping the extra `MaybeLocal<Value>` variable